create_xlinkdocs.sh

```diff
@@ -9,15 +9,26 @@ source "${SCRIPT_DIR}/settings.cfg.sh"
 # shellcheck source=../lsstsw/bin/setup.sh
 source "${LSSTSW}/bin/setup.sh"
 
+print_error() {
+    >&2 echo -e "$@"
```

Comment:
In general I've grown to prefer printf to echo, mostly for not having to worry about different line-ending conventions on different systems.  Just something to consider.